Source/BSGConnectivity.m

```diff
@@ -105,14 +109,21 @@ + (void)monitorURL:(NSURL *)URL usingCallback:(BSGConnectivityChangeBlock)block
         return;
     }
 
+    // The CI environment synthesizes incorrect reachability changes.  Mocking SCNetworkReachability
+    // is the correct way to get round this for testing but until then we explicitly check whether
+    // we're being run under test conditions.
```

Comment:
```suggestion
    // The CI environment synthesizes incorrect reachability changes. If we're under test conditions we want
    // to avoid actually registering for changes - instead they are simulated by calling the callback block directly
```